dockstore-webservice/pom.xml

```diff
@@ -605,7 +605,7 @@
 
     <build>
         <plugins>
-	    <plugin>
+	        <plugin>
```

Comment:
Nitpicky whitespace typo probably

Reply:
I don't actually see this locally, it is just the one tab.

Reply:
@agduncan94  Might be a tab vs spaces thing